bitnami/supabase/templates/studio/tls-secret.yaml

```diff
@@ -3,7 +3,7 @@ Copyright Broadcom, Inc. All Rights Reserved.
 SPDX-License-Identifier: APACHE-2.0
 */}}
 
-{{- if .Values.studio.ingress.enabled }}
+{{- if and .Values.studio.enabled .Values.studio.ingress.enabled }}
```

Comment:
If you make this change, you need to remove the extra `{{- end}}` later in the code